var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -173,7 +173,8 @@ class Vtk(CMakePackage):
     depends_on("gl2ps", when="@8.1:")
     depends_on("gl2ps@1.4.1:", when="@9:")
     depends_on("proj@4", when="@8.2.0")
-    depends_on("proj@4:", when="@9:")
+    depends_on("proj@4:7", when="@9.0")
+    depends_on("proj@8:", when="@9.1:")
```

Comment:
I think this needs to be 9.2, 9.1 still vendors proj@4.4

Reply:
Fixed